README.md

```diff
@@ -10,62 +10,49 @@ new JavaScript features and syntax, so Node.js version 6 or newer is required.
 
 `npm install zipkin --save`
 
-## Usage:
-
-```javascript
-const {tracer, consoleTracer} = require('zipkin');
-
-tracer.pushTracer(consoleTracer);
-```
 
 ## Instrumentations
 
 Various Node.js libraries have been instrumented with Zipkin support.
-In this project:
+Every instrumentation has an npm package called zipkin-instrumentation-*.
 
-- express
-- cujojs/rest
+At the time of writing, zipkin-js instruments these libraries:
 
-## Transports
-
-You can choose between multiple transports; for now,
-scribe and kafka transports are implemented. They can
-be used like this:
+- cujojs/rest (zipkin-instrumentation-cujojs-rest)
+- express (zipkin-instrumentation-express)
+- fetch (zipkin-instrumentation-fetch)
+- memcached (zipkin-instrumentation-memcached)
 
-Scribe:
+Every module has a README.md file that describes how to use it.
 
-```javascript
-const ScribeTracer = require('zipkin-transport-scribe');
-tracer.pushTracer(new ScribeTracer({
-  host: 'localhost',
-  port: 9410
-});
-```
+## Transports
```

Comment:
note we've been using the name "reporter" for the client-side in latest arch docs, though maybe late to change..